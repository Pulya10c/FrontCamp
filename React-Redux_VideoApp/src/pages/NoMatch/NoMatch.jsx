import React from "react";

const NoMatch = () => "Sorry, Invalid address.";

export default NoMatch;
