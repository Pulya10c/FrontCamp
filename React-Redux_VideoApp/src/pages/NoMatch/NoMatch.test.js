import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from './NoMatch';

describe('Component NoMatch', () => {
  it('should render correct message', () => {
    expect(shallow(<NoMatch/>)).toMatchSnapshot();
  });
});
