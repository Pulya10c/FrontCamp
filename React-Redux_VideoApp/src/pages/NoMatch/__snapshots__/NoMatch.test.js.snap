// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component NoMatch should render correct message 1`] = `"Sorry, Invalid address."`;
